tests/with_both_options.py

```diff
@@ -0,0 +1,40 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from unittest import main, TestCase
+
+
+class Test(TestCase):
+    def test_mdx_both_options(self):
+        # from markdown import markdown
+        # from mdx_unimoji import UnimojiExtension
+        # img_heart = '<img alt="love" src="heart.png"/>'
+        # img_tongue = '<img alt=":P" src="tongue.png"/>'
+        # overrides = UnimojiExtension.EMOJI
+        # overrides.update({
+        #     img_heart: ['<3'],
+        #     img_tongue: ':p :P :-p :-P'.split(),
+        # })
+        # text = 'I <3 you! Just kidding. :P'
+        # should_be = '<p>I <img alt="other" src="heart.png" /> you! Just kidding. <img alt=":P" class="other" src="tongue.png" /></p>'
+        # result = markdown(
+        #     text,
+        #     extensions=[UnimojiExtension(span_class='other', emoji=overrides)])
+        # self.assertEqual(should_be, result)
```

Comment:
This test never passes, but I included it commented out so others might be able to get it to pass. I'll remove it.